src/interfaces/comment-interface.ts

```diff
@@ -1,8 +1,14 @@
 /** A comment from the API. */
 export interface CommentIF {
-  businessId?: string;
-  comment: string;
-  filingId?: string;
-  submitterDisplayName: string;
-  timestamp: string;
+  businessId?: string
+  comment: string
+  filingId?: string
+  submitterDisplayName: string
+  timestamp: string
+}
+
+export const EmptyComment: CommentIF = {
```

Comment:
It's nice to have this kind of object to initialize things (or provide prop defaults).